Azeem/src/DLMFtex2Wiki.py

```diff
@@ -156,42 +157,42 @@ def DLMF(n):
             if "\\section" in line:
                 parse=True
                 secCounter+=1
-                wiki.write("drmf_bof\n")
-                wiki.write("\'\'\'"+KLS.secLabel(KLS.getString(line))+"\'\'\'\n")
-                wiki.write("{{DISPLAYTITLE:"+(sections[secCounter][0])+"}}\n")
-                wiki.write("<div id=\"drmf_head\">\n")
-                wiki.write("<div id=\"alignleft\"> << [["+KLS.secLabel(sections[secCounter-1][0])+"|"+KLS.secLabel(sections[secCounter-1][0])+"]] </div>\n")
-                wiki.write("<div id=\"aligncenter\"> [[Zeta_and_Related_Functions#"+
+                KLS.append_text("drmf_bof\n")
```

Comment:
@pwang00 I merged this PR to move forward. Please remove the bof and eof statements, and replace them with the append revision function from the KLS program.